pkg/watcher/watcher.go

```diff
@@ -24,22 +23,27 @@ import (
 
 	hubblev1 "github.com/cilium/hubble/pkg/api/v1"
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	containerIDLen = 15
 	containerIdx   = "containers-ids"
+	podAddrIdx     = "pods-addrs"
 )
 
 var (
 	errNoPod = errors.New("object is not a *corev1.Pod")
 )
 
+type PodFilter struct {
```

Comment:
I wanted to keep single method to find pod, but now this struct is not clear. Would it be fine if I add comment that you can search only by containerID or addr but not both?

Reply:
I agree to have separate method.

Reply:
Sounds good! I merged #555 which will probably create some conflicts for you. Hopefully nothing too hard, but let me know if I can help with rebasing. Thanks!